src/wallet/rpcwallet.cpp

```diff
@@ -210,7 +229,7 @@ UniValue getrawchangeaddress(const UniValue& params, bool fHelp)
 
     CKeyID keyID = vchPubKey.GetID();
 
-    KeyIO keyIO(Params());
+    KeyIO keyIO(chainparams);
     return keyIO.EncodeDestination(keyID);
```

Comment:
This function (`getrawchangeaddress`) should be altered to use the change address chain under the legacy transparent account, instead of drawing from the keypool (which as currently modified just generates external-chain addresses). We can fix this at the same time as we figure out what we are doing with the keypool; add a reference to here in that issue.

Reply:
I've added a note to https://github.com/zcash/zcash/issues/5360